packages/components/src/mobile/bottom-sheet/cell.native.js

```diff
@@ -323,7 +322,7 @@ class BottomSheetCell extends Component {
 		const opacity =
 			activeOpacity !== undefined
 				? activeOpacity
-				: get( platformStyles, 'activeOpacity.opacity' );
+				: platformStyles.activeOpacity?.opacity;
```

Comment:
`platformStyles` is already being used as an object in this component, so using `platformStyles.activeOpacity` is safe. We're keeping the optional chaining for `.opacity` since it is not certain that it will exist.